### PR TITLE
Fix printf File Name too Long error

### PR DESCRIPTION
**Pull request by karlding**
Fix the `File Name too Long` error.

### CODE REVIEW

File: tests/base64_filename_too_long_like_actually_way_too_long_beyond_reasonable_length_that_files_should_naturally_be_test.in [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+export EMAILS=""
+export MAIL_SCRIPT="$RUN_DIR/src/disrupt.sh"
+export FACTS_SOURCE="$TEST_DIR/base64_filename_too_long_like_actually_way_too_long_beyond_reasonable_length_that_files_should_naturally_be_test.lf"
```
Comment (Caelink) [inline tests/base64_filename_too_long_like_actually_way_too_long_beyond_reasonable_length_that_files_should_naturally_be_test.in]: Do we know the limits of printf with %s? I agree this is probs fine, just wondering

File: tests/run_all.sh [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 
 export TEST_DIR="$PWD/tests"
-tests_to_run=(img_test img_test_png fake_image_test base_test url_test)
+tests_to_run=(
+    img_test
+    img_test_png
+    fake_image_test
+    base_test
+    url_test
+    base64_filename_too_long_like_actually_way_too_long_beyond_reasonable_length_that_files_should_naturally_be_test
```
Comment (Caelink) [inline tests/run_all.sh:10]: kek

File: src/disrupt.sh [hunk 3]
```diff
@@ -31,7 +31,7 @@ if [[ "${DICE_ROLL}" -eq 1 ]]; then
     export INCLUDE_IMAGE=1
 
     # Choose a random image from the image assets directory
-    RANDOM_IMAGE=$(printf '%q' "${IMAGES_LIST}" | shuf -n 1)
+    RANDOM_IMAGE=$(printf '%s' "${IMAGES_LIST}" | shuf -n 1)
```
Comment (Caelink) [inline src/disrupt.sh:34]: Is there documentation on max length of %s? As long as the unit test names are more than whatever android and facebook files are named usually it should be fine tho

File: tests/base64_filename_too_long_like_actually_way_too_long_beyond_reasonable_length_that_files_should_naturally_be_test.out [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+--body--
+--message
+Content-Type: image/jpeg; name="lucashi.jpg"
+Content-Disposition: inline; filename="lucashi.jpg"
+Content-Transfer-Encoding: base64
+Content-ID: <lucas>
+X-Attachment-Id: lucas
+
+/9j/4AAQSkZJRgABAQEASABIAAD//gBNRmlsZSBzb3VyY2U6IGh0dHA6Ly9jb21tb25zLndpa2lt
```
Comment (Caelink) [inline tests/base64_filename_too_long_like_actually_way_too_long_beyond_reasonable_length_that_files_should_naturally_be_test.out:28]: you have 2 test images with ridiculously long file names?

**Review by Caelink — APPROVED** [review]
Seems well tested, minor comments and curiosities that might be useful to write down somewhere

File: tests/base64_filename_too_long_like_actually_way_too_long_beyond_reasonable_length_that_files_should_naturally_be_test.out (hunk 4 above)
Comment (karlding) [reply tests/base64_filename_too_long_like_actually_way_too_long_beyond_reasonable_length_that_files_should_naturally_be_test.out:28]: Yeah, they're just the same image, so their `base64` representation will be the same.

**Pull request merged**